exampleCourse/questions/workshop/PeriodicTable1/question.html

```diff
@@ -1,6 +1,6 @@
 <pl-question-panel>
-  <p> {{params.name}} has density $\rho = {{params.rho}} \rm g/cm^3$ and atomic weight $M = {{params.M}} \rm g/mol $</p>
-  <p> Determine the atomic number $N$.</p>
+  <p> {{params.name}} has density $\rho = {{params.rho}} \rm g/cm^3$ and a molar mass $M = {{params.M}} \rm g/mol $</p>
+  <p> Determine the number of atoms in 1 cm<sup>3</sup>.</p>
```

Comment:
The LaTeX formatting with `$ ... $` ensures consistent formatting with the previous paragraph. I added the text "of this element" to remove any ambiguity.

```suggestion
  <p> Determine the number of atoms in $1 \rm cm^3$ of this element.</p>
```